feeds.conf.default

```diff
@@ -4,5 +4,7 @@ src-git routing https://github.com/openwrt-routing/packages.git
 src-git telephony https://github.com/openwrt/telephony.git
 src-git management https://github.com/openwrt-management/packages.git
 src-git targets https://github.com/openwrt/targets.git
+src-git pkg https://github.com/CreatorDev/openwrt-feeds.git
```

Comment:
This won't be accepted upstream. They won't add new feeds. If any of those packages are generally useful, they should be upstreamed to the main openwrt feed. (the ca8210 driver for example?)